pdf-email/functions/pdf.js

```diff
@@ -0,0 +1,111 @@
+/**
+ *
+ *  This Function shows you how to create and send pdf on the fly using Twilio Sendgrid.
+ *  The main idea is that files created on the fly within a Twilio Function are always stored under the /tmp/ directory
+ *  So after the pdf making, we grant the relevant permissions to the file so that we can access the contents and send it via email as normal
+ *  IMPORTANT: Use this function if you want to perform an action that does not require permanent storage as the contents of /tmp/ are removed - preferably single function invocation logic
+ *
+ *  Pre-requisites
+ *  - You need to have a Twilio Sendgrid API KEY
+ *  - You need to include the following npm modules: pdfmake, fs, pdf-to-base64, request, node-cmd
+ */
+
+exports.handler = function(context, event, callback) {
+  /*Details to be passed. Here they are passed statically, in real scenario you pass it as parameters in the Twilio Function*/
+  let description = event.Body || 'Content of the pdf';
+
+  /* change this to a dynamic value if needed*/
+  const recipientEmailAddress = context.TO_EMAIL_ADDRESS;
+
+  var pdfPrinter = require('pdfmake');
+  var fs = require('fs');
+
+  /*Prepare the body of the pdf. For more info on options, look at pdfmake doc
+      http://pdfmake.org/#/
+      */
+  var docDefinition = {
+    content: [
+      { text: '\n' + title, fontSize: 18 },
```

Comment:
Where did this `title` variable come from?

Reply:
It was from the previous version but I had it simplified and forgot to remove it from here. Thanks